src/modm/platform/adc/stm32/adc_impl.hpp.in

```diff
@@ -97,6 +97,12 @@ modm::platform::Adc{{ id }}::setChannel(const Channel channel,
 	ADC{{ per }}->SQR3 = uint32_t(channel) & 0x1f;
 
 	setSampleTime(channel, sampleTime);
+%% if temperature_available
```

Comment:
maybe add the thing with the temperature and internal ref to the commit message :)

Reply:
what do you mean rephrase the commit message?

Reply:
yep, exactly!

Reply:
Just did it